tests/validation/tests/v3_api/test_import_rke2_cluster.py

```diff
@@ -1,16 +1,20 @@
+from time import sleep
```

Comment:
Let's remove this since we aren't using it anymore

Reply:
Sure, removed 💯